src/custom/pages/Claim/InvestmentFlow/index.tsx

```diff
@@ -102,27 +103,32 @@ function _calculateTotalVCow(allClaims: ClaimWithInvestmentData[]) {
 }
 
 type AccountDetailsProps = {
+  index: number
   label: string
   account: string
   connectedAccount: string
-  Icon: Icon
 }
 
-function AccountDetails({ label, account, connectedAccount, Icon }: AccountDetailsProps) {
+function AccountDetails({ index, label, account, connectedAccount }: AccountDetailsProps) {
   return (
-    <span>
-      <b>
-        <Icon width={14} height={14} /> {label}:
-      </b>
-      <i>
-        <ExplorerLink id={account} label={account} type={ExplorerDataType.ADDRESS} />{' '}
+    <div>
+      {index === 0 && (
```

Comment:
Using the suggestion above, here we'd have:

```suggestion
      {isClaimer && (
```